### DISCUSSION TITLE
only the Local Settings option is showing up

### DISCUSSION THREAD

**Discussion by nmoon88** [JVital2013/vitality-goes#71] [Q&A] [answered]
At the moment despite supposedly having all the directories configured correctly I am only seeing the Local Settings and System Information options on the web page that the server generates. I followed the instructions in the readme file involving the syntax but am still having problems. The files are being dumped by goestools through goesproc with SatDump being used as the de modulator onto an external hard drive the location of which is known. Am I missing something or is there some sort of weirdness with external hard drives?

**Comment by JVital2013** [comment, marked as answer]
Answered in #74
